packages/widgets/src/menubar.ts

```diff
@@ -417,17 +423,20 @@ export class MenuBar extends Widget {
     // Fetch the key code for the event.
     let kc = event.keyCode;
 
-    // Do not trap the tab key.
+    // Reset the active index on tab (but not reverse tab), but do not trap the tab key.
     if (kc === 9) {
+      if (!event.shiftKey) {
+        this.activeIndex = -1;
```

Comment:
Sorry, even with the comment I'm still not quite sure I get what this line is doing. 

I thought if it's not resetting the `activeIndex`, then I should be able to do the following on the example page:

1. `Tab` to the menu bar
2. Right arrow key over to Edit (it's highlighted now)
3. Click on the text area input ("Edit" is still highlighted)
4. Press `Shift` + `Tab` and...
    **Expected** behavior: "Edit" remains highlighted and if I press `Enter` it should open the Edit menu. 
    **Actual** behavior: the File menu items becomes highlighted and if I press `Enter` it opens the File menu.